### PR TITLE
Made all requested changes. removed red background, added Hover effect and active effect on menu items.

### PR DESCRIPTION
**Pull request by gvirus21**
# Description

Added hover and active effect to the menu items.

## Fixed #(issue_no)

fixed Red background on sidebar animation.

### CODE REVIEW

**Review by bislara — COMMENTED** [review]
Can you explain the changes in package-lock.json?

**Comment by gvirus21** [comment]
Actually i didn't changed anything in packet.json it's showings this after i did a npm install in my environment,i don't know why. you can manually check if there is any.

**Comment by bislara** [comment]
> Actually i didn't changed anything in packet.json it's showings this after i did a npm install in my environment,i don't know why. you can manually check if there is any.

Could you please ignore the changes in that file and commit? only commit the changes in the CSS file.

File: src/components/NavBar/NavBar.css [hunk 1]
```diff
@@ -1,90 +1,93 @@
...
+  position: relative;
+  font-size: 1.2em;
+  text-transform: capitalize;
+  letter-spacing: 0.4px;
+  text-decoration: none;
+  color: #fff;
+  font-family: Montserrat;
+  font-style: normal;
+  font-weight: normal;
+  padding: 1em;
```
Comment (bislara) [inline src/components/NavBar/NavBar.css:77]: change the padding to 0.7em. The gap seems more with respect to the font size

**Comment by bislara** [comment]
@gvirus21 Let me know when you are done. Also, You raised two PRs. Close one of that.

**Comment by gvirus21** [comment]
Hey sorry about those PRs, i am new to contribution so i am having some issues. But yes hopefully i will be done by tomorrow.

**Comment by bislara** [comment]
No issues bro. If you get any doubt, let me know about it too, would love to help out.

**Pull request closed without merging**